jwst/regtest/test_nirspec_lamp_fs_spec2.py

```diff
@@ -19,21 +19,16 @@ def run_pipeline(rtdata_module):
     # Run the calwebb_spec2 pipeline; save results from intermediate steps
     args = ["jwst.pipeline.Spec2Pipeline", rtdata.input,
             "--steps.assign_wcs.save_results=true",
-            "--steps.msa_flagging.save_results=true",
```

Comment:
Which output? The results from this fixture are in the test itself, `test_nirspec_lamp_fs_spec2`.

Reply:
I guess the appropriate question might have been "was this flag unnecessary to begin with?" then - removing the save_results flag for this step had me thinking that a comparison to an msa_flagging output would no longer be possible.

Reply:
Yes, that argument was unnecessary, because the pars-spec2pipeline ref file skips the msa_flagging step for fixed-slit exposures, hence there was no output to save. So it was a dead line of code.